src/otx/core/model/anomaly.py

```diff
@@ -157,6 +155,22 @@ def configure_callbacks(self) -> list[Callback]:
             ),
         ]
 
+    def on_validation_epoch_start(self) -> None:
+        """Don't call OTXModel's ``on_validation_epoch_start``."""
+        return
```

Comment:
Does it make sense to raise an error here, since this method should never be called?

Reply:
Some callbacks in check if `on_validation_epoch_start` has been overridden, and if so, run the required step. But in this case it might not be necessary. I'll revisit again.